rust/blockstore/src/types/reader.rs

```diff
@@ -93,8 +93,8 @@ impl<
         key_range: KeyRange,
     ) -> Result<Vec<(K, V)>, Box<dyn ChromaError>>
     where
-        PrefixRange: RangeBounds<&'prefix str> + Clone + 'referred_data,
-        KeyRange: RangeBounds<K> + Clone + 'referred_data,
+        PrefixRange: RangeBounds<&'prefix str> + Clone,
+        KeyRange: RangeBounds<K> + Clone,
```

Comment:
was unnecessarily over-specified (I think I had just copied it from the above function def, where this constraint *is* needed)